pkg/sawmill/sawmill.go

```diff
@@ -0,0 +1,86 @@
+package sawmill // If you're wondering why this package is named sawmill, I do too: https://clips.twitch.tv/PlacidOutstandingPelicanCharlieBitMe
```

Comment:
[nit]

Sorry I am a fan of comments above things and not inline. Can you move this to be above the `package sawmill` line?

Reply:
Sure, no problem

Reply:
AH of course haha. My bad! Can you move it to the line above the import?

Reply:
Sure!